proofs/cbmc/.gitignore

```diff
@@ -1,3 +1,5 @@
+# Copyright (c) The mlkem-native project authors
+# Copyright (c) The mldsa-native project authors
 # SPDX-License-Identifier: Apache-2.0
```

Comment:
Why is this one `Apache-2.0`

Reply:
This is from mlkem-native where we have forgotten it. Let's change it there first.